h/dataservice.h

```diff
@@ -66,10 +67,13 @@ typedef struct DataService_tag {
   JsonObject *serviceDefinition;
   uint64 loggingIdentifier;
   WebPlugin *plugin;
+  struct Storage_tag *storage;
 } DataService;
 
 WebPlugin *makeWebPlugin(char *baseDir, struct JsonObject_tag *pluginDefintion, InternalAPIMap *internalAPIMap,
                          unsigned int *idMultiplier, int pluginLogLevel);
+WebPlugin *makeWebPlugin2(char *pluginLocation, JsonObject *pluginDefintion, InternalAPIMap *internalAPIMap,
+                         unsigned int *idMultiplier, int pluginLogLevel, struct Storage_tag *storage);
```

Comment:
Can you document the arguments (i.e. doxygen style)? It's unclear why a web plugin needs some storage.